### DISCUSSION TITLE
REQ: maintainance page

### DISCUSSION THREAD

**Discussion by punkyard** [nextcloud/all-in-one#4197] [Ideas] [answered]
hi!

I wanted to bring here the subject of the 502 page when containers are being upgraded

<img width="1090" alt="no maintenance mode AiO" src="https://github.com/nextcloud/all-in-one/assets/59349105/be9a6128-4dd4-4059-9bfa-f9a5b29fa8f9">

Would it be possible to expose a 'maintainance page' during the time of the updates and upgrades?
that would look good ✌️ thanks

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/3945
